sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/GraphClient.java

```diff
@@ -32,14 +34,20 @@
 public class GraphClient {
     private static final Logger LOGGER = LoggerFactory.getLogger(GraphClient.class);
 
+    public static final String GROUP_ID = "GROUP_ID";
+
+    public static final String GROUP_NAME = "GROUP_NAME";
+
     private final AADAuthenticationProperties properties;
 
     public GraphClient(AADAuthenticationProperties properties) {
         this.properties = properties;
     }
 
-    public Set<String> getGroupsFromGraph(String accessToken) {
-        final Set<String> groups = new LinkedHashSet<>();
+    public Map<String, Set<String>> getGroupsFromGraph(String accessToken) {
```

Comment:
1. Define a new class named `GroupInformation`, Use `GroupInformation` instead of `Map<String, Set<String>>` as return type.
2. Rename method to `getGroupInformation`